schema/factory/CommonFactory.go

```diff
@@ -28,18 +28,22 @@ import (
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
 	"github.com/th2-net/th2-common-go/schema/common"
+	"github.com/th2-net/th2-common-go/schema/metrics"
 )
 
 const (
-	configurationPath = "/var/th2/config/"
-	jsonExtension     = ".json"
-	CUSTOM_FILE_NAME  = "custom"
+	configurationPath    = "/var/th2/config/"
+	jsonExtension        = ".json"
+	CUSTOM_FILE_NAME     = "custom"
+	PROMETHEUS_FILE_NAME = "prometheus"
 )
 
 type CommonFactory struct {
 	modules     map[common.ModuleKey]common.Module
 	cfgProvider ConfigProvider
 	zLogger     zerolog.Logger
+	liveness    *metrics.AggregatingMetricMonitor
+	prometheus  *metrics.PrometheusServer
```

Comment:
Could you please move Prometheus related code to the PrometheusModule, and register it automatically